generators/app/templates/.storybook/webpack.config.js

```diff
@@ -1,3 +1,5 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
```

Comment:
The Arrange component added this in eslintrc, do we want to update the generator to do the same considering it's repeated several times? Or would it be dangerous to make a global assumption, masking a potential real case the lint should throw this error?

https://github.com/cerner/terra-arrange/pull/2/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R15
https://github.com/cerner/terra-arrange/pull/2/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R15